src/gui/widgets/template_list_widget.cpp

```diff
@@ -784,66 +787,42 @@ void TemplateListWidget::updateButtons()
 	delete_button->setEnabled(single_template_selected);	/// \todo Make it possible to delete multiple templates at once
 	move_up_button->setEnabled(single_row_selected && !first_row_selected);
 	move_down_button->setEnabled(single_row_selected && !last_row_selected);
-	edit_button->setEnabled(single_template_selected);
 	
-	bool georef_visible = false;
-	bool georef_active  = false;
-	bool custom_visible = false;
-	bool custom_active  = false;
-	bool import_active  = false;
-	if (mobile_mode)
-	{
-		// Leave most buttons invisible
-		edit_button->setVisible(false);
-	}
-	else if (single_template_selected)
+	if (!mobile_mode)
 	{
-		auto temp = map->getTemplate(posFromRow(visited_row));
-		auto is_georeferenced = temp->isTemplateGeoreferenced();
-		auto is_visible = template_table->item(visited_row, 0)->checkState() == Qt::Checked;
+		// Update and show other buttons
 		
-		georef_action->setChecked(is_georeferenced);
-		georef_visible = true;
-		georef_active = is_visible && temp->canChangeTemplateGeoreferenced();
-		
-		custom_visible = !is_georeferenced || temp->canChangeTemplateGeoreferenced();
-		custom_active = !is_georeferenced && is_visible;
+		bool is_georeferenced = false;
+		bool edit_enabled   = false;
+		bool georef_enabled = false;
+		bool custom_enabled = false;
```

Comment:
`custom_` is a very generic name prefix. How about `position_change_` or something better expressing that the variable controls visibility of template position controls?